assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -47,10 +47,20 @@ const BoxedNumeric = styled(Numeric)`
   border-radius: 4px;
 `;
 
+const StyledToggle = styled(Toggle)`
+  + span {
+    margin: 0 10px;
+  }
+`;
+
 function isNum(v) {
   return typeof v === 'number' && !isNaN(v);
 }
 
+function formatRotationAngleNumber(v) {
+  return v > 360 ? v % 360 : v;
```

Comment:
Just `v % 360` is enough.

Reply:
It's a good point on the iterative editing. We could pass the value as is, but instead use `usePresubmitHandler` to do `x % 360` to "normalize" it for submit.

Reply:
That will mean that the `pushUpdate` could simply be: `pushUpdate({rotationAngle: value})`. Which might also be simpler.

Reply:
So we should allow the user input 1000, and only on submit, get 1000 % 360 (280), right? Just push this case using `usePresubmitHandler`.

Reply:
![Kapture 2020-04-02 at 0 59 46](https://user-images.githubusercontent.com/1680157/78209438-62783300-747d-11ea-88df-3d7860f798e8.gif)

Reply:
Yup looks good!